tests/_test_mscolab/test_files.py

```diff
@@ -220,7 +220,7 @@ def _example_data(self):
           </ListOfWaypoints>
       </FlightTrack>"""
 
-    def _create_operation(self, flight_path="firstflight", user=None, content=None):
```

Comment:
that is correct

but you need for the case that it is None use the xml from the stub_code because now to have to initialize an operation by sending data. Which was in the past done on the server.